aptitude.sh

```diff
@@ -44,7 +44,7 @@ bootstrap_cmake(){
 build_deps(){
     ## need to account for debian
     compiler_pkgs="gcc g++"
-    if [[ "$OS" = Ubuntu* ]]; then
+    if [[ "$OS" = Ubuntu* && "$OS_MAJOR" -lt 22 ]]; then
```

Comment:
are these required changes for a successful build on 22.04? if so, an accompanying ci job could ensure that the build stays successful

Reply:
They are not required for a successful build. We just no longer install dependencies from 3rd party repos that are recent enough in the official ones.